test: added unit tests for Circle.java

Tornadoguy26 opened the pull request:
# PULL_REQUEST_TEMPLATE

Description of issue and request.
- Added CircleTest.java (testcases)

## Type of change

- [ ] Bug fix.
- [x] New feature.
- [ ] Modified feature.
- [ ] Deleted feature.
- [x] Testing.
- [ ] Documentation.

## Checklist:

- [x] Code performs functionality of the project.
- [x] Code does not generate any errors or warnings.
- [x] Added testcases and tested code.
- [x] Performed a self-review of the the code.
- [x] Contacted Partners to merge and review code.

RamyPoe committed:
chore: add circle tests

RamyPoe committed:
fix: function name typo

RamyPoe commented:
```
index 0000000..ba97aea
--- /dev/null
+++ b/Circle.java
@@ -0,0 +1,18 @@
+public class Circle {
+       Point Center;
+       double radius;
+       
+       public Circle(Point Center, double radius) {
+               this.Center = Center;
diff --git a/Circle.java b/Circle.java
new file mode 100644
index 0000000..ba97aea
--- /dev/null
+++ b/Circle.java
@@ -0,0 +1,18 @@
+public class Circle {
+       Point Center;
+       double radius;
+       
+       public Circle(Point Center, double radius) {
+               this.Center = Center;
+               this.radius = radius;
+       }
+       
+       public double area(){
+               return Math.PI * Math.pow(radius, 2);
+       }
+       
+       public double circumference() {
+               return 2 * Math.PI * radius;
+       }
+
+}
\ No newline at end of file
diff --git a/TestCircle.java b/TestCircle.java
deleted file mode 100644
index d4b4446..0000000
--- a/TestCircle.java
+++ /dev/null
@@ -1,25 +0,0 @@
-package lab4;
-
-import static org.junit.Assert.*;
-
-import org.junit.Test;
-
-public class TestCircle {
-       
-       private static final double ERR = 0.000001;
-
-       @Test
-       public void testCircleArea() {
-               Point a = new Point(0, 0);
-               Circle circ = new Circle(a, 1);
-               assertEquals(circ.area(), Math.PI, ERR);
-       }
-
-       @Test
-       public void testCircleCircumference() {
-               Point a = new Point(0, 0);
-               Circle circ = new Circle(a, 0.5);
-               assertEquals(circ.circumference(), Math.PI, ERR);
-       }
-       
-}

```

Tornadoguy26 commented:
Merged (compare) test/Rameen/Circle into (base) feature/Tornadoguy26/Circle to include TestCircle within the base branch.

**git diff HEAD..origin/test/Rameen/Circle 
Output:**
```bash
diff --git a/Circle.java b/Circle.java
deleted file mode 100644
index ba97aea..0000000
--- a/Circle.java
+++ /dev/null
@@ -1,18 +0,0 @@
-public class Circle {
-       Point Center;
-       double radius;
-
-       public Circle(Point Center, double radius) {
-               this.Center = Center;
-               this.radius = radius;
-       }
-
-       public double area(){
-               return Math.PI * Math.pow(radius, 2);
-       }
-
-       public double circumference() {
-               return 2 * Math.PI * radius;
-       }
-
-}
\ No newline at end of file
diff --git a/TestCircle.java b/TestCircle.java
new file mode 100644
index 0000000..d4b4446
--- /dev/null
+++ b/TestCircle.java
@@ -0,0 +1,25 @@
+package lab4;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class TestCircle {
+
+       private static final double ERR = 0.000001;
+
+       @Test
+       public void testCircleArea() {
+               Point a = new Point(0, 0);
+               Circle circ = new Circle(a, 1);
+               assertEquals(circ.area(), Math.PI, ERR);
+       }
+
+       @Test
+       public void testCircleCircumference() {
+               Point a = new Point(0, 0);
+               Circle circ = new Circle(a, 0.5);
+               assertEquals(circ.circumference(), Math.PI, ERR);
+       }
+
+}
```

**git diff HEAD..origin/feature/Tornadoguy26/Circle 
Output:**
```sh
diff --git a/Circle.java b/Circle.java
new file mode 100644
index 0000000..ba97aea
--- /dev/null
+++ b/Circle.java
@@ -0,0 +1,18 @@
+public class Circle {
+       Point Center;
+       double radius;
+
+       public Circle(Point Center, double radius) {
+               this.Center = Center;
+               this.radius = radius;
+       }
+
+       public double area(){
+               return Math.PI * Math.pow(radius, 2);
+       }
+
+       public double circumference() {
+               return 2 * Math.PI * radius;
+       }
+
+}
\ No newline at end of file
diff --git a/TestCircle.java b/TestCircle.java
deleted file mode 100644
index d4b4446..0000000
--- a/TestCircle.java
+++ /dev/null
@@ -1,25 +0,0 @@
-package lab4;
-
-import static org.junit.Assert.*;
-
-import org.junit.Test;
-
-public class TestCircle {
-
-       private static final double ERR = 0.000001;
-
-       @Test
-       public void testCircleArea() {
-               Point a = new Point(0, 0);
-               Circle circ = new Circle(a, 1);
-               assertEquals(circ.area(), Math.PI, ERR);
-       }
-
-       @Test
-       public void testCircleCircumference() {
-               Point a = new Point(0, 0);
-               Circle circ = new Circle(a, 0.5);
-               assertEquals(circ.circumference(), Math.PI, ERR);
-       }
-
-}
```
